app/config/EADConfig.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import features.Feature.Feature
+import features.{Feature, FeatureStatus}
+import features.FeatureStatus.FeatureStatus
+import javax.inject.{Inject, Singleton}
+import play.api.Configuration
+
+@Singleton
+class EADConfig @Inject()(configuration: Configuration) {
```

Comment:
Nit: Didn't we have agreement not to capitalise the whole acronym, but only the first letter?

Reply:
You're right! Changed